### DISCUSSION TITLE
Horizontal scrollbar is only visible on scrolling till the bottom

### DISCUSSION THREAD

**Discussion by ag288** [chakra-ui/chakra-ui#7599] [Q&A]
I have a table component, that has quite a few columns, and thus requires horizontal scrolling. However, the horizontal scroll bar is visible only when we scroll right to the bottom of the table, which is very user-unfriendly. Is there a way of making the horizontal scroll bar appear on the bottom of the screen always, so that the user can scroll to the right without having to scroll down till the bottom of the table?

**Comment by barkow96** [comment]
Have you found any solution on this?

**Comment by ag288** [comment]
Nope no hope
